api/Dockerfile

```diff
@@ -96,6 +98,7 @@ RUN useradd --create-home opener \
 USER opener
 
 # Copy code into the final image
+COPY --chown=opener --from=packages openverse-attribution /py-packages/openverse-attribution/
```

Comment:
This shouldn't be necessary, I thought, because it should be copied into the `.venv` for production (non-editable). Unless this is explicitly to support the development-mode of this target? I wish it were possible to conditionally copy, if that is the reason.

Bit of a nit, though, we can iterate on this without issue.

Edit (after reading the compose file again): Even in development mode, it shouldn't be necessary, because we map the python packages directory in the compose file, so the editable path reference should still be valid.